### PR TITLE
[fix] Always filter out the ALL schema_grants

### PR DESCRIPTION
**Pull request by edulop91**

### CODE REVIEW

File: pkg/resources/view_grant_acceptance_test.go [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/stretchr/testify/require"
 )
 
 func TestAccViewGrantBasic(t *testing.T) {
-	vName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	viewName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
```
Comment (alldoami) [inline pkg/resources/view_grant_acceptance_test.go]: Can combine viewName and databaseName to same value all in one line? just a suggestion

File: pkg/resources/view_grant_acceptance_test.go [hunk 2]
```diff
@@ -38,48 +35,44 @@ func TestAccViewGrantShares(t *testing.T) {
...
-				ImportStateVerify: true,
-			},
 		},
 	})
 }
 
 func TestAccFutureViewGrantChange(t *testing.T) {
-	vName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	viewName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
```
Comment (alldoami) [inline pkg/resources/view_grant_acceptance_test.go]: same here?

File: pkg/resources/view_grant_acceptance_test.go (hunk 1 above)
Comment (alldoami) [reply pkg/resources/view_grant_acceptance_test.go]: viewName, databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)

Comment (edulop91) [reply pkg/resources/view_grant_acceptance_test.go]: that won't work though since `acctest.RandStringFromCharSet` has a single string return value

Comment (alldoami) [reply pkg/resources/view_grant_acceptance_test.go]: ah I see, I thought there'd be a way to give it the same value. Forgot that's how go interprets that syntax though

**Pull request merged**